### ISSUE TITLE
Mirroring PRs fails due to SSH error

### ISSUE THREAD

**Issue by ePirat** [brndnmtthws/labhub#63]
I've already opened an issue with git2 as I guess it's an issue with either that library or the libssh2 that it's using but maybe this is some known issue and can be worked around somehow or I just did something wrong… (Although googling for that error it seemed that it is caused by a bug in libssh2)

When I trigger a PR mirror it fails with the following error:

```
labhub[929]: Error: Caught error handling PR: GitError { message: "Git error: \"Failed to authenticate SSH session: Unable to send userauth-publickey request\" …
```

**Comment by ePirat** [comment]
Spent some more time investigating this, it fails when trying to push to gitlab, keeps calling the key callback over and over until it fails.
(Note: I've added the "Creating key with key path" message to verify it uses the right key file)
```
--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:67
Updated tips, ref=refs/tags/20190129 oid1=0000000000000000000000000000000000000000 oid2=200a55e2e0473411eecdbddce0b0aa5794018e29

--> src/github.rs:67
Updated tips, ref=refs/tags/20190205 oid1=0000000000000000000000000000000000000000 oid2=2cec0f9aca5b9c079fc5967c886910fb2f3aa1c9

--> src/github.rs:67
Updated tips, ref=refs/tags/20190212 oid1=0000000000000000000000000000000000000000 oid2=8eae88fcda424c6cf0819102c1b7faaecde53d08

--> src/github.rs:67
Updated tips, ref=refs/tags/20190219 oid1=0000000000000000000000000000000000000000 oid2=bd1d986e51b8c8606c68f06ff0e246b158ce6aa1

--> src/github.rs:67
Updated tips, ref=refs/tags/20190309 oid1=0000000000000000000000000000000000000000 oid2=0a35174f959269c147a497e7ac4ee8843f393107

--> src/github.rs:67
Updated tips, ref=refs/tags/20190406 oid1=0000000000000000000000000000000000000000 oid2=bfc0837ae55a14894fc2d0c7d1eaf90a493064ed

--> src/github.rs:67
Updated tips, ref=refs/tags/20190430 oid1=0000000000000000000000000000000000000000 oid2=ef7a66a2ff710352d8fd0048c67048158c2e28ff

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/bugfix/build-grep oid1=0000000000000000000000000000000000000000 oid2=d40304b3a13b7f5165e31c5ab35e36a513f37e8a

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/bugfix/non-fatal-distclean oid1=0000000000000000000000000000000000000000 oid2=2c97c0fcb0ebc17ed3a19456bfcfb9c0dd8984c8

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/bugfix/reference oid1=0000000000000000000000000000000000000000 oid2=b42905636adadce4cfcacfe6bca7ed3a9ac94c56

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/cdef-rebase-f5890e3 oid1=0000000000000000000000000000000000000000 oid2=aef01444ad5e5091817105adefcf99d1e54c3058

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/cleanup/no-wx oid1=0000000000000000000000000000000000000000 oid2=91af948fdbd26b2f86bd87ac6ae3ec0bf1c86bb6

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/cleanup/sort-mv-weights oid1=0000000000000000000000000000000000000000 oid2=fe0a1e866611b288d31f786cb6e73078ff57f49a

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/cleanup/std-ln2 oid1=0000000000000000000000000000000000000000 oid2=a74919a34007143c5d29810514c5270fb1b6a38d

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/cosmetics/ec-simplify-loop oid1=0000000000000000000000000000000000000000 oid2=a3e0fc489e5775a4cecacc131ed134129f2884fb

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/experiment/cdef-off oid1=0000000000000000000000000000000000000000 oid2=21bfd4e5406d12ee7a8feb578b9314dec15336c0

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/feature/cdef oid1=0000000000000000000000000000000000000000 oid2=36f563655df291d73b4c695c2eb2793851b11bb2

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/fix-submodule oid1=0000000000000000000000000000000000000000 oid2=77c288fecc3237a1162b5ece2218dc714de2bb37

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/fix_left_coeff_context oid1=0000000000000000000000000000000000000000 oid2=f01674d76b77175278d1ccc03cea467dc8fd0d84

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/inline-always oid1=0000000000000000000000000000000000000000 oid2=0af9deb6cb11c9e68016e382d564daaec35a336a

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/ivf-subcrate oid1=0000000000000000000000000000000000000000 oid2=631ef14506e58fee319b9a25d4196a97dec2d76d

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/master oid1=0000000000000000000000000000000000000000 oid2=16ea18ea6eaf3f0e8549a6804ebb45ea37f659fb

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/optional_rec oid1=0000000000000000000000000000000000000000 oid2=3f179a8520d5b6a71807fbcf05f13c7deac32bd5

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/pick_cdef_from_q oid1=0000000000000000000000000000000000000000 oid2=d523fd9322aa7574f9db939096827a14a5d19378

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/revert-134-update10 oid1=0000000000000000000000000000000000000000 oid2=413a0741c5c4bcf10c5c7447fda7ee7be0db045f

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/revert-408-mc1 oid1=0000000000000000000000000000000000000000 oid2=a60e41e1c9ab75d1daf1c030b8af535b279f4b63

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/revert-518-mc10 oid1=0000000000000000000000000000000000000000 oid2=8a78032b2c94dfc96e3ebbcc4cca6fb13cab0ec5

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/revert-53-smooth_hv oid1=0000000000000000000000000000000000000000 oid2=ec86b2e505c076e75001175c5a16b166c0a7cc57

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/revert-54-refactor_prediction oid1=0000000000000000000000000000000000000000 oid2=9fb9effa7649bb370a8bba820e88f20c4a347c38

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/revert-613-idct oid1=0000000000000000000000000000000000000000 oid2=207667f23e874ec125cb87e0a290bdd1599f9f05

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/revert-856-cdef_fast oid1=0000000000000000000000000000000000000000 oid2=4a9b63ac28767c57acccbbc8d2ddde7de9232293

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/revert_refactor oid1=0000000000000000000000000000000000000000 oid2=07f532e02fcfd712a408bcd1c9b3910b23772405

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/rzumer-patch-1 oid1=0000000000000000000000000000000000000000 oid2=2f5cfd3d014d1c29f9cb3dbe22a2345024da82d4

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/rzumer-patch-2 oid1=0000000000000000000000000000000000000000 oid2=ff96ca62c5f7b60ebfb56fe3ef5c9b17356c4f58

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/rzumer-patch-3 oid1=0000000000000000000000000000000000000000 oid2=7d01b985ceecf407e7ba4d52cbd95525bb27d583

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/rzumer-patch-4 oid1=0000000000000000000000000000000000000000 oid2=e6b3b55c6bb1fdec70817cbb97b840087431d701

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/rzumer-patch-5 oid1=0000000000000000000000000000000000000000 oid2=de386c761d3e2665e9f83d249ac3cfa0b1e2cf49

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/sec-tap oid1=0000000000000000000000000000000000000000 oid2=b7c9b24336a55e4dd4fdbb434e817b315f235888

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/smaller-tables oid1=0000000000000000000000000000000000000000 oid2=66e40830bd5417c2ed2fa8953e4d1d5c9def2e7c

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/smooth_hv oid1=0000000000000000000000000000000000000000 oid2=3e737f515980e5f0fdd1629e5893f0a140318241

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/split_rdo oid1=0000000000000000000000000000000000000000 oid2=cc2c477d20b22617f55e46ccc0fc49ce079326d6

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/tdaede-patch-1 oid1=0000000000000000000000000000000000000000 oid2=eccdee63d62e159d30c05ae4c1773da27595638c

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/tdaede-patch-2 oid1=0000000000000000000000000000000000000000 oid2=4d5320e79d2af2d33cff96db74f5ac5e267d78ff

--> src/github.rs:67
Updated tips, ref=refs/remotes/origin/var_tx_2 oid1=0000000000000000000000000000000000000000 oid2=97375cb8d1fa3f1ea3242a7feead56279db89d14
Cloned new repo git@github.com:xiph/rav1e.git into /tmp/.tmpm85qyx
handle_pr_updated_with_repo
pr_handle=PrHandle {
    base_full_name: "xiph/rav1e",
    head_full_name: "KyleSiefring/rav1e",
    github_remote: "github-1396",
    gitlab_remote: "gitlab",
    gitref: "iimg_lrf_review",
    github_clone_url: "git@github.com:KyleSiefring/rav1e.git",
    pr_number: 1396,
}
Fetching remote=github-1396 ref=iimg_lrf_review

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:67
Updated tips, ref=refs/remotes/github-1396/iimg_lrf_review oid1=0000000000000000000000000000000000000000 oid2=c2a3a01a0543f3a26848bbbf670bd36e6ac349bc
Successfully fetched remote

--> src/github.rs:187
Creating ref refs/heads/pr-1396/KyleSiefring/rav1e/iimg_lrf_review from refs/remotes/github-1396/iimg_lrf_review, id=c2a3a01a0543f3a26848bbbf670bd36e6ac349bc
Pushing PR remote=gitlab ref=iimg_lrf_review number=1396 base_full_name=xiph/rav1e

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"

--> src/github.rs:47
Entered Git credential callback, cred=SSH_KEY | SSH_MEMORY | SSH_CUSTOM

--> src/github.rs:52
Creating key with key path="/usr/local/etc/labhub.key"
Error: Caught error handling PR: GitError { message: "Git error: \"Failed to authenticate SSH session: Unable to send userauth-publickey request\"" }
```

**Comment by ePirat** [comment]
After debugging this for waaaay too long it turned out it is a bug in labhub, at https://github.com/brndnmtthws/labhub/blob/master/src/github.rs#L150

It hardcodes gitlab.com instead of honoring the configured host.
